src/base/bittorrent/session.cpp

```diff
@@ -3087,6 +3103,42 @@ void Session::setIPFilterFile(const Path &path)
     }
 }
 
+QStringList Session::excludedFileNames() const
+{
+    return m_excludedFileNames;
+}
+
+void Session::setExcludedFileNames(const QStringList &excludedFileNames)
+{
+    if (excludedFileNames != m_excludedFileNames)
+    {
+        m_excludedFileNames = excludedFileNames;
+        populateExcludedFileNamesRegExpList();
+    }
+}
+
+void Session::populateExcludedFileNamesRegExpList()
+{
+    m_excludedFileNamesRegExpList.clear();
+
+    for (const QString &str : asConst(excludedFileNames()))
+    {
+        const QString pattern = QRegularExpression::anchoredPattern(QRegularExpression::wildcardToRegularExpression(str));
+        const QRegularExpression re{ pattern, QRegularExpression::CaseInsensitiveOption };
```

Comment:
```suggestion
        const QRegularExpression re {pattern, QRegularExpression::CaseInsensitiveOption};
```

Reply:
It had correct coding style in [suggestion](https://github.com/qbittorrent/qBittorrent/pull/17106#discussion_r884253778) above.